ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -48,6 +52,15 @@ def _get_group(self, base_url, group_name):
         except Exception, e:
             raise e
 
+    def _get_organization(self, base_url, org_name):
+        url = base_url + self._get_action_api_offset() + '/organization_show?id=' + org_name
+        try:
+            content = self._get_content(url)
+            content_dict = json.loads(content)
+            return content_dict['result']
+        except Exception, e:
```

Comment:
I understand the reasoning, but here it really doesn't matter why it fails. The implementation is analog to the implementation of `_get_group`, but of course it could be changed in both places. If - as you wrote - the parsing of the JSON failed, it should fail. Maybe it's better to remove the try/catch entirely, and delegate the exception handling to the caller.

In this particular case the [caller catches all exception in an `except` block](https://github.com/metaodi/ckanext-harvest/blob/120-create-remote-orgs/ckanext/harvest/harvesters/ckanharvester.py#L353), as any exception simply means, that the remote organization could not be fetched.

What do you think?